spaces/templates/post_edit.html

```diff
@@ -116,4 +116,122 @@ <h2 >Box Your Post</h2>
+                </div>
+
+                <div class="mb-3">
+                    <label  for="tags" class="form-label">Tags:</label>
+                    <input name="tags" type="Tags" class="form-control" id="exampleTags" aria-describedby="TagsHelp">
+                    <div id="TagsHelp" class="form-text">Provide a Tag</div>
+                </div>
+
+                <br>
+
+                <p>
+                    <label  for="id_upload">Upload Image:</label>
+                    <br/>
+                    <br/>
+                    <input type="file" name="image" id="id_upload">
+                </p>
+                <br>
+                <p>
+                    <label  for="id_text">Text:</label>
+                    <br/>
+                    <textarea name="text" cols="40" rows="10" required="" id="id_text" data-gramm="false"
+                            wt-ignore-input="true"></textarea>
+                </p>
+
+<p>
+    <label for="id_spaces">Select spaces:</label>
+    <select name="spaces" id="id_spaces" multiple>
+        {% for space in spaces %}
+            {% if space.id in form.initial.spaces %}
+                <option value="{{ space.id }}" selected>{{ space.name }}</option>
+            {% else %}
+                <option value="{{ space.id }}">{{ space.name }}</option>
+            {% endif %}
+        {% endfor %}
+    </select>
+</p>
+
+
+
+                <button style="background-color: rgba(245, 194, 68, 1); color:black" type="submit" class="btn btn-primary">Save</button>
+                {% if form.errors %}
+                        <ul>
+                            {% for error in form.errors %}
+                                <li>{{ error }}</li>
+                            {% endfor %}
+                        </ul>
+                {% endif %}
+
+            </form>
+    </div>
+    </div>
+
+<script>
+$(document).ready(function() {
+    $('#link-input').on('change', function() {
+        var linkValue = $(this).val();
+        $.ajax({
+            url: '/check_link/',
+            data: {
+                'link': linkValue
+            },
+            dataType: 'json',
+            success: function(response) {
+                if (response.exists) {
+                    // If the error message is displayed and the user updates the link
+                    if ($('#link-error').length) {
+                        $('#link-error').remove(); // Remove the error message
+                    }
+                    $('#link-input').after('<div id="link-error" class="alert alert-warning" role="alert">This link already exists in the database. You may update the link.</div>');
+                } else {
+                    // Removal of error message when link does not exist in database
+                    if ($('#link-error').length) {
+                        $('#link-error').remove();
+                    }
+                }
+            }
+        });
+    });
+});
+</script>
+
+
+
```

Comment:
Why do we have the lines from 119-236? Can you please check this wrong duplication of code? @cemrekut